shared/wallets/transaction/index.tsx

```diff
@@ -74,7 +74,11 @@ type CounterpartyTextProps = {
 export const CounterpartyText = (props: CounterpartyTextProps) => {
   switch (props.counterpartyType) {
     case 'airdrop':
-      return <Text style={{color: globalColors.white}} type={props.textTypeSemibold}>Stellar airdrop</Text>
+      return (
```

Comment:
I ran into a similar autoformatting thing in #18214, think your merge conflicts are probably from that

Reply:
:+1: Fixed the merge conflict